tests/test_flask_admin.py

```diff
@@ -82,6 +82,7 @@ def test_add_distro(self):
                 sess['email'] = 'pingou@pingoured.fr'
 
             output = c.get('/distro/add', follow_redirects=True)
+            print output.data
```

Comment:
Is this something left over from testing, or should this now always be in the tests?